.github/workflows/cargo.yml

```diff
@@ -1,47 +1,99 @@
 on:
   push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
 
 env:
   CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
 
 jobs:
-  linux:
-    name: ubuntu
-    runs-on: ubuntu-20.04
+  build:
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macos-10.15 ]
+        profile: [ release, debug ]
+    name: build-${{ matrix.os }}-${{ matrix.profile }}
+    runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
-    - name: Run
-      run: bash .github/workflows/ci.sh
-  macos:
-    name: macos
-    runs-on: macos-10.15
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - name: Update cargo flags
```

Comment:
Do we need to set `RUST_BACKTRACE` env variable somewhere?

Reply:
I moved it to the global environment (line 9).